app/assets/javascripts/browsered/worldpay-3ds-flex-ddc.js

```diff
@@ -3,6 +3,23 @@
 const toggleWaiting = () => {
   document.getElementById('card-details-wrap').classList.toggle('hidden')
   document.getElementById('spinner').classList.toggle('hidden')
+
+  var paymentMethodSubmitElement = document.getElementById('payment-method-submit')
+  if (typeof paymentMethodSubmitElement !== 'undefined' && paymentMethodSubmitElement !== null) {
+    paymentMethodSubmitElement.classList.toggle('hidden')
+  }
+
+  var payDividerElements = document.getElementsByClassName('pay-divider')
+  if (typeof payDividerElements !== 'undefined' && payDividerElements.length > 0) {
+    payDividerElements[0].classList.remove('pay-divider')
+    payDividerElements[0].classList.toggle('hidden')
+  }
+
+  var errorSummaryContainer = document.getElementById('error-summary')
+  if (typeof errorSummaryContainer !== 'undefined' && errorSummaryContainer !== null &&
+    !errorSummaryContainer.classList.contains('hidden')) {
```

Comment:
As we can't guarantee the current `hidden` state of these elements, should we just use the `add` method in favour of `toggle`? That way we can't accidentally 'unhide' something that was supposed to stay hidden. 

It would mean we wouldn't have to check for the existence of hidden and make this something like: 
```js
document.getElementById('error-summary').classList.add('hidden')
```

Reply:
👍 Updated https://github.com/alphagov/pay-frontend/commit/d943f2e416bf5ebac36e5704e328d19bab257854